kitting_trays:
  tray_ids: [1, 2, 3, 4, 5, 6] 
  slots: [1, 2, 3, 4, 5, 6]
